internal/version/version.go

```diff
@@ -0,0 +1,9 @@
+package version
+
+var (
+	// Version holds the complete version number. Filled in at linking time.
+	Version = "dev"
+
+	// GitCommit holds the git revision. Filled in at linking time.
+	GitCommit = "unknown"
```

Comment:
It might be unnecessary for now, but we might also consider storing the build date for improved testability on local or CI builds.

Reply:
TBH, I thought `GitCommit` is already a bit much. I'll skip date for now. 👍